tests/test_dois.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from invenio.testsuite import InvenioTestCase, make_test_suite, run_test_suite
+from dojson.contrib.marc21.utils import create_record
+from inspirehep.dojson.hep import hep
+from inspirehep.dojson.utils import strip_empty_values
+
+
+class DoisTests(InvenioTestCase):
+
+    def setUp(self):
```

Comment:
Since these fixtures are used exactly once each there's no reason to share them among all tests. Just add them to their respective test, so that the fixture being tested and the code that uses it are near each other.